GitHub_LearningTests/Feature/Dashboard/Presentation/DashboardPresenterTests.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  DashboardPresenterTests.swift
+//  GitHub_LearningTests
+//
+//  Created by Amin Ashhab on 14.12.2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import Foundation
+import XCTest
+@testable import GitHub_Learning
+
+class DashboardPresenterTests: XCTestCase {
+    private var sut: DashboardPresenterImpl!
+    
+    private func setupTests(
+        dashboardCotroller: DashboardController = DashboardControllerDummy(),
+        logOutController: LogOutController = LogOutControllerDummy(),
+        dashboardView: DashboardView = DashboardViewDummy()
+    ) {
+        sut = DashboardPresenterImpl(dashboardCotroller: dashboardCotroller, logOutController: logOutController)
+        sut.view = dashboardView
+    }
+    
+    func test_givenDashboardView_whenViewDidLoad_thenSetEmailIsCalled() {
+        // Given
+        let view = DashboardViewDummy()
+        setupTests(dashboardView: view)
+        
+        // When
+        sut.viewDidLoad()
+        
+        // Then
+        XCTAssert(view.isCalled == true)
+    }
```

Comment:
Tento test je zbytecny, ale klidne ho tam nech. Ten pod tim testuje uplne to same.